README.md

```diff
@@ -346,6 +346,28 @@ Contributions and suggestions for new features are welcome, as are bug reports!
 Please create a new [issue](https://github.com/ewels/rich-click/issues)
 or better still, dive right in with a pull-request.
 
+### Local setup
+
+1. Create a new venv with a python3.6+ interpreter using `python3 -m venv venv`
+2. Activate the venv with `source venv/bin/activate`
+3. Install our the package as an editable including all dev dependencies with `pip3 install -e ."[dev]"`
+4. Install pre-commit with `pre-commit install`
+
+#### Pre-commit
+
+Our pre-commit hooks contain the following hooks:
+
+- prettier: formats our yaml files nicely.
+- no relative imports: prevents you from using relative imports.
+- iSort: will automatically sort the imports alphabetically.
+- black: will automatically format your code to be according to standardized python format.
+- flake8: will do linting checks to make sure all your code is correctly styled and used.
+- mypy: static type checker which verifies you are not using objects incorrectly.
+
+As mentioned, some of these tools automatically fix your code while other only highlight potential issues.
+Therefore, sometimes it enough to try to commit a second time, and it will pass, while others will require
+manual changes to your code.
+
 ## Credits
```

Comment:
Too far off the bottom of the diff to add a suggestion in the PR, but you should add yourself!

Reply:
> Oops, sorry about the ❌ It's amazingly difficult to write valid markdown in the GitHub PR plain text editor 😆
> 
> Note that for next time you can batch suggestions together if you want instead of doing one commit per comment. This option is only available on the _Files changed_ PR tab though, not _Overview_.

Haha let me just do it on my macbook then :p